sql/catalyst/src/main/scala/org/apache/spark/sql/connector/expressions/aggregates.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.connector.expressions
+
+import org.apache.spark.sql.types.DataType
+
+case class Min(column: FieldReference) extends AggregateFunc
+
+case class Max(column: FieldReference) extends AggregateFunc
```

Comment:
`Min` and `Max` should have the same data type as the original column, so no data type is needed here.

Reply:
We can also document the data type of `Sum`:
1. return long for integral types
2. return double for float/double
3. return `decimal(p + 10, s)` for `decimal(p, s)`